### ISSUE TITLE
Feature request: Create branch or repo using fluentd driver instead of tailing logs

### ISSUE THREAD

**Issue by ryanwalls** [SumoLogic/fluentd-kubernetes-sumologic#9]
Would love to see an example where containers on a k8s cluster are set to use the fluentd logging driver.

**Comment by stevezau** [comment]
I'm not sure this will work as we require the container ID to query kubernetes metadata from the master. Does that exist in the fluentd logging driver?

**Comment by ryanwalls** [comment]
@stevezau Yep, see https://docs.docker.com/engine/admin/logging/log_tags.  Maybe could also use the k8s downward api, https://kubernetes.io/docs/user-guide/downward-api instead of querying the k8s api.

**Comment by stevezau** [comment]
It could work, just need to be able to work with https://github.com/fabric8io/fluent-plugin-kubernetes_metadata_filter

we are open to a PR if you want to give it a go?

**Comment by ryanwalls** [comment]
@stevezau In the middle of a big push at work, but will keep it on my "to do" list for later this month.
